core/src/main/java/org/zenoss/zep/dao/impl/EventSummaryDaoImpl.java

```diff
@@ -228,7 +228,14 @@ public String create(Event event, final EventPreCreateContext context) throws Ze
         final byte[] fingerprintHash;
         final String uuid;
         if (ZepConstants.CLOSED_STATUSES.contains(event.getStatus())) {
-            fingerprintHash = DaoUtils.sha1(fingerprint + '|' + System.currentTimeMillis());
+            long ts;
+            ts = System.currentTimeMillis();
+            // Change fingerprintHash value for clear event to prevent
+            // undesirable DuplicateKeyException (ZEN-26244)
+            if (event.getSeverity() == EventSeverity.SEVERITY_CLEAR) {
```

Comment:
Talked to @fraibacas and he explained. Further down in this method events that are being clared  are updated with a fingerprint that is generated by also using `System.currentTimeMillis()`.  Subtracting 1 from the value here ensures that the same timestamp isn't used.  Please add a comment to this effect.